modules/cactus-web/src/Accordion/Accordion.tsx

```diff
@@ -751,7 +753,9 @@ const variantStyles = (props: AccordionProps): ReturnType<typeof css> | undefine
   }
 }
 
-export const Accordion = styled(AccordionBase)`
+export const Accordion = styled(AccordionBase).withConfig(
+  omitProps<AccordionProps>('flex', 'flexBasis', 'flexGrow', 'flexShrink')
```

Comment:
You can just pass the styled-system function here. In fact, you can also pass `margin`, `'width'`, and `'maxWidth'` and get rid of the `omitMargins` call on line 682. (Not a big deal if you don't, though.)
```suggestion
  omitProps<AccordionProps>(flexItem, margin, 'width', 'maxWidth')
```

Reply:
Ah, yeah the styled-system function didn't exist yet at the time that I made these changes. I'll update it.